ur_bringup/launch/ur_moveit.launch.py

```diff
@@ -20,6 +20,7 @@
 from ament_index_python.packages import get_package_share_directory
```

Comment:
We should at some point test creating `xml`-launch files . This here are getting really nasty.

Reply:
It is not that bad. Also I am not sure how many examples are there about xml launch files. We started to catch the grip with the python-based one :)